framework/source/class/qx/event/handler/DragDrop.js

```diff
@@ -575,6 +576,7 @@ qx.Class.define("qx.event.handler.DragDrop",
 
         this.__manager.addListener(this.__root, "pointermove", this._onPointermove, this);
         this.__manager.addListener(this.__root, "pointerup", this._onPointerup, this, true);
+        this.__manager.addListener(this.__root, "track", this._onPointermove, this);
       }
     },
 
```

Comment:
please elaborate on adding "track"

Reply:
Its http://www.qooxdoo.org/devel/api/#qx.event.type.Track where track is derived from http://www.qooxdoo.org/devel/api/#qx.event.type.Pointer only adding one additional method ```getDelta``` which is documented as:
Returns a map with the calculated delta coordinates and axis, relative to the position on trackstart event.